### PR TITLE
Mongodb added

### PR DESCRIPTION
**Pull request by DLiSWE**
Update package.json, data source and configurations for mongodb for chat.

### CODE REVIEW

**Comment by DLiSWE** [comment]
Package.lock is messed up, will fix

File: package.json [hunk 1]
```diff
@@ -17,18 +17,22 @@
...
     "envalid": "^7.3.1",
-    "express": "^4.18.1",
+    "express": "^4.18.2",
     "helmet": "^5.1.1",
     "hpp": "^0.2.3",
     "jsonwebtoken": "^8.5.1",
+    "kerberos": "^2.0.1",
+    "mongodb": "^3.7.3",
     "morgan": "^1.10.0",
+    "package.json": "^2.0.1",
```
Comment (raylu511) [inline package.json:35]: Remove this

File: src/app.ts [hunk 2]
```diff
@@ -59,7 +59,15 @@ class App {
 
   private async initializeDataSource() {
     try {
-      await Datasource.initialize();
+      await pgDataSource.initialize();
+      console.log('pg init')
```
Comment (raylu511) [inline src/app.ts:63]: lets remove this console.log

File: src/app.ts [hunk 3]
```diff
@@ -59,7 +59,15 @@ class App {
...
+    } catch (error) {
+      console.error(error);
+    }
+
+    try {
+      await mongoDataSource.initialize();
+      console.log('mg init')
```
Comment (raylu511) [inline src/app.ts:70]: and remove this one

File: src/entities/index.ts [hunk 4]
```diff
@@ -1,2 +1 @@
-export { default as UserEntity } from './users.entity';
-export { default as CommentEntity } from './comments.entity';
+export { PostEntity, CommentEntity, UserEntity } from './sql'
```
Comment (raylu511) [inline src/entities/index.ts:1]: newline at the end of file

**Review by ZohaibManzoor00 — APPROVED** [review]
Looks like you configured MongoDb in a very similar way to Postgres. 
Overall looks fine

**Pull request merged**